server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/burstcoin/BurstExtension.java

```diff
@@ -31,10 +31,14 @@
 
 public class BurstExtension implements IExtension{
     @Override
-    public String getName() { return "BATM Burstcoin extension"; }
+    public String getName() {
```

Comment:
We should leave the return for the extension name, the reason for this is one of the test's is to see if it loads. YOu will see a message about each loaded extension....

Reply:
It's the same, I just expanded the method

Reply:
Eh either works, i was only keeping a consistency with baseline that was started.